### ISSUE TITLE
Choose components to use

### ISSUE THREAD

**Issue by fernandodof** [akveo/blur-admin#67]
Hello everyone,

Is there a way to choose which components I want to use ? for example: I don't want to use "Chartist".
If so how can I do that ? It would make easier to integrate to an existing project.

**Comment by lugovsky** [comment]
You can just delete that component from project and dependency from bower. Let me know if you have any more questions

**Comment by fernandodof** [comment]
Alright, I haven't thought of that. I will try it and let you know. 

Thanks.

**Comment by fernandodof** [comment]
@lugovsky I think I din't quite explained my question. There some comoponents/libs that I don't need in my project, so I would like to remove them front the project. Is there an easy way to do that ? If remove them from bower the project still requires it, doesn't it ?

Thank you.

**Comment by vazh** [comment]
if you remove it from bower, it removes it from gulp-injection into html, in this case into index.html.
but you still need to remove the module declaration in angular. just use project widesearch and then remote it.

**Comment by lugovsky** [comment]
@fernandodof Yeah, @vazh is absolutely right. Unfortunately you still need to remove angular module dependencies. For example, if you don't need Chartist you will need to remove

- angular-chartist.js and chartist
- `angular-chartist` dependency from `theme.module.js` file
- `BlurAdmin.pages.charts.chartist` from `charts.module.js`

Please refer to angular module documentation for additional information: https://docs.angularjs.org/guide/module
